src/glewlwyd.h

```diff
@@ -32,17 +32,28 @@
 #include <jansson.h>
 #include <jwt.h>
 
-#define _GNU_SOURCE
-#define __USE_GNU
+#ifndef _GNU_SOURCE
+ #define _GNU_SOURCE
+#endif
+
+#ifndef __USE_GNU
+ #define __USE_GNU
+#endif
+
```

Comment:
I tried to create a new pull request. Somehow, I am unable to create one. I am trying again.